src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -271,15 +272,39 @@ public FeedbackSessionsCopyFromModal getCopyFromModal() {
             }
         }
         
-        // Add "No active courses" option if there are no active courses
-        if (result.isEmpty()) {
-            ElementTag blankOption = createOption("No active courses!", "", true);
-            result.add(blankOption);
+        // if courses is not empty but result is empty, means the instructor have no permission to edit
+        // rather than the instructor have no active courses
+        // add "No active courses" option if there are no active courses
+        ElementTag defaultOption;
+        if (result.isEmpty() && !courses.isEmpty()) {
+            defaultOption = createOption("No permission to modify courses!", "", true);
+            result.add(defaultOption);
+        } if (result.isEmpty()) {
```

Comment:
A new issue for settling on a proper message in those cases?
What's wrong with the lack of permission to modify the courses' sessions, specifically?